### PR TITLE
Add support of yarn installed by system package manager

### PR DESCRIPTION
**Pull request by ungarscool1**
In response of my issue #6 I've writed the code to allow yarn installed by system package manager.

### CODE REVIEW

File: install-yarn.js [hunk 1]
```diff
@@ -6,13 +6,9 @@ var yarnBin
   , node = process.argv[0]
   ;
 
-if (process.env['npm_execpath'] && process.env['npm_execpath'].match(/node_modules[\/\\]yarn[\/\\]bin[\/\\]yarn\.js$/)) {
+if (process.env['npm_execpath'] && process.env['npm_execpath'].match(/[\/\\]yarn(-.+)?[\/\\]bin[\/\\]yarn\.js$/)) {
```
Comment (ayan4m1) [inline install-yarn.js]: What is the `(-.+)?` looking for here?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "install-peers",
-  "version": "1.0.3",
+  "version": "1.0.4",
```
Comment (ayan4m1) [inline package.json]: The package version should be managed by the maintainers, imo.

File: install-yarn.js (hunk 1 above)
Comment (ungarscool1) [reply install-yarn.js]: As the path can be `/opt/yarn-v1.22.17/bin/yarn.js`, I add an optional group to append the `-vx.xx.xx`

File: install-yarn.js [hunk 3]
```diff
@@ -6,13 +6,9 @@ var yarnBin
   , node = process.argv[0]
   ;
 
-if (process.env['npm_execpath'] && process.env['npm_execpath'].match(/node_modules[\/\\]yarn[\/\\]bin[\/\\]yarn\.js$/)) {
+if (process.env['npm_execpath'] && process.env['npm_execpath'].match(/[\/\\]yarn(-v\d+\.\d+\.\d+)?[\/\\]bin[\/\\]yarn\.js$/)) {
   var execPath = process.env['npm_execpath'];
-  var expectedPath = path.join('yarn', 'bin', 'yarn.js');
```
Comment (ayan4m1) [inline install-yarn.js:11]: Can't you modify this to `path.join('bin', 'yarn.js');` and leave the rest of the code the way it is?

Comment (ungarscool1) [reply install-yarn.js:11]: if you want, but it's the same thing as the regex

Comment (ayan4m1) [reply install-yarn.js:11]: The thing is I'm not sure what the goal of

```
if (execPath.slice(-1 * expectedPath.length) === expectedPath) {
```

is and I don't want to introduce a regression by removing it.

**Pull request merged**